tests/microos/verify_setup.pm

```diff
@@ -0,0 +1,299 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Basic check for ignition/combustion setup
+# Maintainer: qa-c team <qa-c@suse.de
+
+use Mojo::Base qw(opensusebasetest);
+use testapi;
+use utils qw(systemctl);
+use YAML::PP;
+use File::Basename qw(basename);
+
+my $data;
+my $fail = 0;
+
+sub load_test_data {
+    my $ypp = YAML::PP->new();
+    $data = $ypp->load_file(sprintf("%s/data/microos/butane/config.fcc", get_var('CASEDIR')));
+    # remove butane metadata
+    delete $data->{version};
+    delete $data->{variant};
+}
+
+sub get_test_object {
+    my ($entry, $test_object) = @_;
+    return $data->{$entry}{$test_object};
+}
+
+sub print_summary {
+    my $tests = shift;
+    my (@errors) = @_;
+
+    if (@errors) {
+        record_info('Fail', join("\n", @errors), result => 'fail');
+        $fail = 1;
+    } else {
+        record_info("OK", "All $tests passed!");
+    }
+}
+
+sub systemd_tests {
+    my $units = get_test_object('systemd', 'units');
+    my @errors = ();
+
+    if ($units) {
+        record_info('systemd', 'Checking system setup');
+    } else {
+        record_info('SKIP', 'Skiping systemd tests!');
+        return;
+    }
+
+    foreach my $unit (@$units) {
+        my $name = $unit->{name};
+        systemctl("is-enabled $name", expect_false => !$unit->{enabled});
+        systemctl("is-active $name", expect_false => ($name =~ /sshd/) ? 0 : 1);
+
+        if (exists $unit->{contents}) {
+            if (script_run("grep 'Just a Test!' /etc/systemd/system/$name") != 0) {
```

Comment:
Shouldn't it be `/etc/systemd/system/$name.service`?

I see that the VRs are passing, I'm just wondering how and why.

Reply:
It contains the unit type extension. It takes from data/microos/butane/config.fcc L#26

```
[2022-09-22T17:33:09.744313+02:00] [debug] <<< consoles::serial_screen::type_string(cmd="backend_type_string", text="grep 'Just a Test!' /etc/systemd/system/create_test_file.service", json_cmd_token="ruTbfOMH")
```

Reply:
Ahhh, I see, thanks!